hw01_hello_otus/go.mod

```
@@ -1,3 +1,5 @@
-module github.com/fixme_my_friend/hw01_hello_otus
+module github.com/frobbery/go_course/tree/master/hw01_hello_otus
```

```suggestion
module github.com/frobbery/go_course/hw01_hello_otus
```
Поправил немного. 
И не забывайте в этом и следующих ДЗ удалять .sync файлы.